tests/puppeteer/campaigns/functional/BO/08_design/07_linkWidget/01_CreateAndCheckFooterLinkWidget.js

```diff
@@ -85,7 +85,7 @@ describe('Create footer link widget and check it in FO', async () => {
       const textResult = await this.pageObjects.addLinkWidgetPage.addLinkWidget(LinkWidgets.demo_1);
       await expect(textResult).to.equal(this.pageObjects.linkWidgetsPage.successfulCreationMessage);
 
-      numberOfLinkWidgetInFooter = await this.pageObjects.linkWidgetsPage.getNumberOfElementInGrid(35);
+      numberOfLinkWidgetInFooter = await this.pageObjects.linkWidgetsPage.getNumberOfElementInGrid(36);
```

Comment:
Should be a const. Will be easier to change in the future ;)

Reply:
Instead of a const, I created a files with hooks : {id, name}, the name is already used in a another file and maybe we will need more hooks in the future.